ansible/roles/go-ipfs-deploy-docker/tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+- name: build go-ipfs Docker image
+  include_tasks:
+    file: build.yml
+    apply:
+      delegate_to: localhost
+
+- name: copy docker image to server
+  copy: src="/tmp/go-ipfs-{{ go_ipfs_version }}.tar" dest="/tmp/go-ipfs-{{ go_ipfs_version }}.tar"
+
+- name: Load image from archive
+  docker_image:
+    name: "go-ipfs"
+    tag: "{{ go_ipfs_version }}"
+    load_path: "/tmp/go-ipfs-{{ go_ipfs_version }}.tar"
+    source: load
+
+- name: Creates ipfs repo dir
+  file:
+    path: "{{ go_ipfs_repo }}"
+    state: directory
+    owner: "{{ deploy_user }}"
+    group: users
+    mode: '0755'
+
+- name: Start go-ipfs container
+  docker_container:
+    name: "go-ipfs"
+    network_mode: host
+    state: started
+    restart_policy : always
+    image: "go-ipfs:{{ go_ipfs_version }}"
+    command: "{{ go_ipfs_cmd }}"
+    user: ipfs
+    env: "{{ go_ipfs_env }}"
+    log_driver: json-file
+    log_options:
+      max-size: 100m
```

Comment:
how many days of logs is this giving us?

Reply:
If `m` means minutes then 0.6 days.

Reply:
I think that's 100MiB. 
It's hard to give an estimate so here is some data:
Looking at venus which is running witth `IPFS_LOGGING=ERROR"` the earliest log is from `2019-07-12` (the time it was originally launched) and current log size is 125KiB